src/ReactNativeSVG.web.ts

```diff
@@ -156,7 +158,7 @@ function parseTransformProp(
       transformArray.push(`matrix(${transform.join(' ')})`);
     } else {
       const stringifiedProps = transformsArrayToProps(
-        transform as TransformsStyle['transform']
+        transform as TransformsStyleArray
```

Comment:
The `Test JS build / build (pull_request)` build crashed without altering that type.